app/views/frontend/noticeboard.html.erb

```diff
@@ -1,2 +1,15 @@
 <h1>Frontend#noticeboard</h1>
 <p>Find me in app/views/frontend/noticeboard.html.erb</p>
+<style>#font18{font-size:18px;}</style>
```

Comment:
Please remove line 1 and 2.
` <h1>Frontend#noticeboard</h1>
  <p>Find me in app/views/frontend/noticeboard.html.erb</p>`